From 0c3a7538d15617e3878f61d21dfbcf34bff8a41e Mon Sep 17 00:00:00 2001
From: easbarbosa <easbarbosa@pm.me>
Date: Thu, 11 Mar 2021 17:24:10 -0300
Subject: [PATCH] bigger fonts

---
 config.def.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config.def.h b/config.def.h
index 6f05dce..8134035 100644
--- a/config.def.h
+++ b/config.def.h
@@ -5,5 +5,5 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "Liberation Mono:pixelsize=25:antialias=true:autohint=true";
 static int borderpx = 2;

 /*
--
2.30.1
